script/make_utils/jupyter.sh

```diff
@@ -15,7 +15,7 @@ WHAT_TO_DO="open"
 
 # Create a list of notebooks with long execution times in order not to consider them when refreshing
 # all notebooks at the same time.
-LONG_EXECUTION_TIMES_NOTEBOOKS=()
+LONG_EXECUTION_TIMES_NOTEBOOKS=("ClassifierComparison.ipynb")
```

Comment:
do we need to do something here ? should we add a fixme then ?

Reply:
Let me remove this now that knn is gone from the comparison notebook.